uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -150,41 +151,16 @@ define([
     };
 
     /**
-     * Loads the leave request day types
-     *
-     * @return {Promise}
-     */
-    vm._loadDayTypes = function () {
-      return OptionGroup.valuesOf('hrleaveandabsences_leave_request_day_type')
-        .then(function (dayTypesData) {
-          vm._dayTypes = _.indexBy(dayTypesData, 'name');
-        });
-    };
-
-    /**
      * Loads the leave requests and calendar
      *
      * @return {Promise}
      */
-    vm._loadLeaveRequestsAndCalender = function () {
-      return LeaveRequest.all({
-        managed_by: vm.contactId,
-        from_date: {
-          from: vm.selectedPeriod.start_date
-        },
-        to_date: {
-          to: vm.selectedPeriod.end_date
-        }
-      }, {}, null, null, false)
-      .then(function (leaveRequestsData) {
-        vm._indexLeaveRequests(leaveRequestsData.list);
-
-        return vm._loadCalendar();
-      })
-      .then(function () {
-        vm.managedContacts = tempContactData;
-        vm._showMonthLoader();
-      });
+    vm._loadLeaveRequestsAndCalendar = function () {
+      return parentCtrl._loadLeaveRequestsAndCalendar.call(vm, 'managed_by', false)
+        .then(function () {
+          vm.managedContacts = tempContactData;
+          vm._showMonthLoader();
```

Comment:
You have this in both child controllers, meaning it should go in the parent controller